### ISSUE TITLE
Paths wrapping

### ISSUE THREAD

**Issue by irvnriir** [L13/vscode-diff#68]
Paths of diff-found files/folders wrap for 1360px-wide VSCode window, and are not relative to the actual one . (i usually use 1120px)

**Comment by L13** [comment]
Hi, the paths in the list view are not wrapping anymore in the next version. This is an optimization for unlimited files otherwise it will not work.

**Comment by irvnriir** [comment]
I don't think that the wrapping is more important than unlimited files, but it definitely a meaningful ergonomics feature .

Thanks for answers .

**Comment by L13** [comment]
Hi, I have implemented virtual scrolling in the list view. It responses now way faster. So I close the issue because path wrapping would make it too complex. I recommend to use Ctrl/Cmd + B which toggles the sidebar view. I use this command if I need more space in the editor or in the diff panel.

**Comment by ilias-7565** [comment]
@L13 i open Sidebar only when need it and use "[Activitus Bar](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.activitusbar)" Extension . Good to hear it works faster, i guess its about huge amount if differences ?

Couldn't check it because if this X) :
![Screenshot](https://user-images.githubusercontent.com/66683424/116817368-8cef6080-ab55-11eb-97bd-23b4ab09042b.png)

**Comment by irvnriir** [comment]
@L13 i open Sidebar only when need it and use "[Activitus Bar](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.activitusbar)" Extension . Good to hear it works faster, i guess its about huge amount if differences ?

Couldn't check it because of this X) :
![Screenshot](https://user-images.githubusercontent.com/66683424/116817368-8cef6080-ab55-11eb-97bd-23b4ab09042b.png)

**Comment by L13** [comment]
I havn't changed anything in the scan process. It seems `l13Diff.abortOnError` is true.

Yes, the view only renders now what you see. Before all elements were added to the DOM. That was the reason why it was so laggy with a huge amount of files. But HTML, CSS and JS does not provide anything to solve this problem, so I had to write my own virtual scrolling. The VS Code Team did this for their views, too. So I had to rewrite and test a lot of stuff.
